packages/keyring-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.5.1]
+### Fixed
+- Fix `addNewAccountForKeyring` for `CustodyKeyring` ([#1694](https://github.com/MetaMask/core/pull/1694))
```

Comment:
This will need to be updated to include https://github.com/MetaMask/core/pull/1702 as well

Reply:
#1702 is a breaking change as well, so we'll need to replace 7.5.1 with 8.0.0

Reply:
Looks like @adonesky1 took care of this